README.md

```diff
@@ -2,26 +2,29 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+### Setup
+
+1. Install the packages
 
 ```bash
-npm run dev
-# or
-yarn dev
```

Comment:
I decided to remove mentions of `yarn` because we only have `package-lock.json` file committed and not `yarn.lock`. I think it will be easier for everybody if only one package manager is used

Reply:
it's not true (3 years already)
https://classic.yarnpkg.com/blog/2018/06/04/yarn-import-package-lock/

Reply:
`yarn import` should [generate yarn.lock file](https://classic.yarnpkg.com/en/docs/cli/import) and since we don't have it committed in the repo, I still think that we don't really encourage yarn to be used